zen/config/config.c

```diff
@@ -67,6 +76,9 @@ zn_config_create(struct toml_table_t *config_table)
 void
 zn_config_destroy(struct zn_config *self)
 {
+  if (self->space_default_app) {
+    free(self->space_default_app);
+  }
```

Comment:
It's ok to do `free(NULL);`.

```suggestion
  free(self->space_default_app);
```